src/js/util/MutationViewsUtil.js

```diff
@@ -43,11 +43,11 @@ var MutationViewsUtil = (function()
 		missense_mutation: {label: "Missense", style: "missense_mutation"},
 		nonsense_mutation: {label: "Nonsense", style: "trunc_mutation"},
 		nonstop_mutation: {label: "Nonstop", style: "trunc_mutation"},
-		frame_shift_del: {label: "FS del", style: "trunc_mutation"},
-		frame_shift_ins: {label: "FS ins", style: "trunc_mutation"},
-		in_frame_ins: {label: "IF ins", style: "inframe_mutation"},
-		in_frame_del: {label: "IF del", style: "inframe_mutation"},
-		splice_site: {label: "Splice", style: "trunc_mutation"},
+		frame_shift_del: {label: "Frame Shift Deletion", style: "trunc_mutation"},
```

Comment:
This also affects the mutation table values, and may cause corresponding column to become larger than usual. It is better to add another field, say longName, and use that value in the MutationInfoPanelView to avoid changing the mutation table values.